src/ExpressiveInstaller/config.php

```diff
@@ -11,13 +11,22 @@
         'zendframework/zend-expressive-twigrenderer'     => '^1.2.1',
         'zendframework/zend-expressive-zendrouter'       => '^2.0',
         'zendframework/zend-expressive-zendviewrenderer' => '^1.2.1',
-        'zendframework/zend-servicemanager'              => '^2.7.3 || ^3.0',
+        'zendframework/zend-servicemanager'              => '^2.7.3 || ^3.2',
     ],
 
     'require-dev' => [
         'filp/whoops'
     ],
 
+    'application' => [
+        'flat' => [
+            '/Resources/src/ConfigProvider.flat.php' => '/src/App/ConfigProvider.php',
```

Comment:
I can understand why they are confusing. It's caused by the copyFile function:

    copy(__DIR__ . $source, $projectRoot . $target);

It uses the relative dir for the source Vs the project root for the target. If we change this so the source is using the project root as well, it becomes more powerful and it can be used to copy files from anywhere if needed.